src/components/DomainItem/DomainItem.js

```diff
@@ -158,20 +147,9 @@ const Label = ({ domain, timeLeft, isOwner }) => {
     text = 'Owner'
   }
 
-  let timeLeftHuman
-
-  if (domain.state === 'Auction' || domain.state === 'Reveal') {
-    timeLeftHuman = humanizeDate(timeLeft)
-  }
-
   return (
     <LabelContainer className="label-container">
       <LabelText>{text}</LabelText>
-      {domain.state === 'Auction' || domain.state === 'Reveal' ? (
```

Comment:
There are some other old auction related codes such as https://github.com/ensdomains/ens-app/blob/5c838b95965125bbabc57ff87ac299bc3ba01d5f/src/components/DomainItem/DomainItem.js#L139 . Do you need them as well?

Reply:
This is true, this could probably be removed. I will make an issue to remove them, but let's not block this PR for this